widget/ui/src/components/Alert/Alert.types.ts

```diff
@@ -8,5 +9,8 @@ export interface PropTypes {
   footer?: ReactNode;
   action?: ReactNode;
   variant?: 'alarm' | 'regular';
-  containerStyles?: CSSProperties;
+  containerStyles?: { [key: string]: string };
+  titleContainerStyles?: { [key: string]: string };
+  titleSize?: BaseSizes;
```

Comment:
I think that would better to accept a component as `title`, instead of adding Typography props to Alert.
Something like this:
```
<Alert 
title={<Typograph ... />}
/>
```

Reply:
Done